dashboard/test/controllers/api/v1/pd/report_controller_base_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class Api::V1::Pd::ReportControllerBaseTest < ::ActionController::TestCase
+  test 'generate_csv' do
+    data = [
+      {organizer_name: 'Organizer1', district: 'district1'},
+      {organizer_name: 'Organizer2', district: 'district2'},
+      {organizer_name: 'Organizer3', district: 'district3'}
+    ]
+
+    expected_csv = [
+      'Organizer Name,District',
+      'Organizer1,district1',
+      'Organizer2,district2',
+      'Organizer3,district3'
+    ].join("\n")
```

Comment:
%q for multiline. I found this to be more readable in this case, because the %q had extra whitespace before each line which would need to be split and stripped:

```ruby
%q(
  Organizer Name,District
  Organizer1,district1
  ...
)

Alternatively I could use a <<eos or a %q with no indent, but again it looks less nice.